features/helpers/zypper.go

```diff
@@ -0,0 +1,84 @@
+package helpers
+
+import (
+	"encoding/xml"
+	"os/exec"
+	"testing"
+
+	"github.com/SUSE/connect-ng/internal/zypper"
+	"github.com/SUSE/connect-ng/pkg/registration"
+	"github.com/stretchr/testify/assert"
+)
+
+type Zypper struct {
+	t *testing.T
+}
+
+func NewZypper(t *testing.T) *Zypper {
+	return &Zypper{
+		t: t,
+	}
+}
+
+func (zypp *Zypper) run(params ...string) []byte {
+	cmd := exec.Command("zypper", params...)
+	result, err := cmd.CombinedOutput()
+
+	if err != nil {
+		assert.FailNow(zypp.t, "Running zypper failed", err)
```

Comment:
Could we print the output on failure? Maybe it will help debugging assert failures.